samples/v1beta1/buildstrategy/buildah/buildah.yaml

```diff
@@ -0,0 +1,231 @@
+---
```

Comment:
For what is this file being added in addition to what we have in https://github.com/shipwright-io/build/tree/main/samples/v1beta1/buildstrategy/buildah ?